web/scripts/rate_all.py

```diff
@@ -36,44 +34,45 @@ def sanitized_games(games):
         elif not (g.result.startswith('W') or g.result.startswith('B')):
             print('unknown result:  ', g)
             pass
+        elif g.date_played is None or g.date_played.timestamp() == 0.0:
+            print('No date played:  ', g)
+            pass
```

Comment:
I was confused by pass, since the intent is to skip this game. I'd use `continue` everywhere just for consistency, even though code-wise it executes identically as pass in this case because the final action is in an `else` branch.